src/views/BrowseView/BrowseView.tsx

```diff
@@ -48,24 +48,26 @@ const BrowseView: React.FC<RouteComponentProps> = () => {
   }
 
   return (
-    <Container>
-      <StyledBackgroundPattern />
-      <Header variant="hero" ref={headerRef}>
-        Browse
-      </Header>
-      <Text variant="h5">Topics that may interest you</Text>
-      <IntersectionTarget ref={targetRef} />
-      <StyledCategoryPicker
-        categories={categoriesData?.categories}
-        loading={categoriesLoading}
-        selectedCategoryId={selectedCategoryId}
-        onChange={handleCategoryChange}
-        isAtTop={inView}
-      />
-      <ErrorBoundary fallback={ErrorFallback}>
-        <StyledInfiniteVideoGrid categoryId={selectedCategoryId || undefined} ready={!!selectedCategoryId} />
-      </ErrorBoundary>
-    </Container>
+    <>
```

Comment:
Do we need this `React.Fragment`? We only return one top-level element (`Container`) so we shouldn't need that or am I missing something?